Resources/Textures/Clothing/Head/Hats/catears.rsi/meta.json

```diff
@@ -14,11 +14,35 @@
       "name": "equipped-HELMET",
       "directions": 4,
       "delays": [
-                [0.2, 0.2, 0.2, 0.2, 0.2],
-                [0.2, 0.2, 0.2, 0.2, 0.2],
-                [0.2, 0.2, 0.2, 0.2, 0.2],
-                [0.2, 0.2, 0.2, 0.2, 0.2]
-            ]
+        [
+          0.2,
+          0.2,
+          0.2,
+          0.2,
+          0.2
+        ],
+        [
+          0.2,
```

Comment:
Things like this is not a formatting error, for example. If we want this to be styled consistently, the delays corresponding to the same direction should be on the same line like it was before.

Reply:
RSIedit outputs delays with all values being a new line each. That being said, it would be nice if RSIedit output delays as a single line